.github/workflows/run_android_ci.py

```diff
@@ -12,6 +12,7 @@
 parser.add_argument('--run_attempt', required=True, help="A unique number for each attempt of a particular workflow run in a repository")
 parser.add_argument('--project_arn', required=True, help="Arn for the Device Farm Project the apk will be tested on")
 parser.add_argument('--device_pool_arn', required=True, help="Arn for device pool of the Device Farm Project the apk will be tested on")
+parser.add_argument('--device_pool', required=True, help="Which device pool is being used for this test")
```

Comment:
It's more for our own convenience when looking at the test runs on device farm itself. Otherwise you'd have to dig into the run itself to determine which device pool was run. It may or may not be mandatory to have a different name but having identically named tests is not ideal in general.